lib/ranked-model.rb

```diff
@@ -69,7 +74,7 @@ def ranks *args
     end
 
     def column_default ranker
-      column_defaults[ranker.name.to_s] if ActiveRecord::Base.connected?
+      column_defaults[ranker.name.to_s] if ranker_connection_class.connected?
```

Comment:
Would it work if we just called `connected?` on the current class? Then we don't have to worry about `ranker_connection_class` at all.

Reply:
on the right receiver, yeah.. rails connection handlers are per thread class attributes